ledger/ledger-common/src/main/java/net/corda/v5/ledger/common/transaction/PrivacySalt.java

```diff
@@ -0,0 +1,23 @@
+package net.corda.v5.ledger.common.transaction;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+import net.corda.v5.base.annotations.DoNotImplement;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Defines a mechanism for implementing a privacy salt.
+ * A privacy salt is required to compute  a nonce per transaction component in order to ensure that an adversary cannot
+ * use brute force techniques and reveal the content of a Merkle-leaf hashed value.
+ */
+@CordaSerializable
```

Comment:
Why is this `@CordaSeriaizable` now? Wasn't the implementation _already_ `@CordaSerializable`, and isn't that enough?

Reply:
@chrisr3 **PrivacySalt.kt** was annotated `@CordaSerializable`, so I did the same here. Shouldn't we annotate on the interface (where applicable) as a general rule? i.e. if it's on the interface, it will apply to any/all implementations.